### PR TITLE
Feature#48 deployment heroku

### LINKED ISSUE #48: Deployment in heroku
# REQUIREMENTS
- Need to change database to postgres when it's on production

# Acceptance Criteria
List all required acceptance criteria
- [ ] Heroku compile it correctly
- [ ] Still works in heroku

### PR DESCRIPTION
**Pull request by AngelBFdev**
# Description
- Now sqlite3 is just for development
- Now Postgres is used in production
- Assets will be compile when production
  
  ## Screenshots 
Project in heroku
image

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -32,6 +31,11 @@ gem 'bootsnap', '>= 1.4.4', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'sqlite3', '~> 1.4'
```
Comment (diegoOG09) [inline Gemfile:34]: Creo que les conviene usar Postgres incluso desde development porque nosotros tuvimos problemas usando sqlite en dev y luego postgres en prod.

**Review by diegoOG09 — APPROVED** [review]
LGTM

File: Gemfile (hunk 1 above)
Comment (AngelBFdev) [reply Gemfile:34]: Que errores? @diegoOG09

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by JuanequeX — COMMENTED** [review]
I will recommend that use Postgres globally, in production, and development, my knowledge about to use two kinds of data base in an project like this ain't the puntual to say that this is correct, they should take a advice with your mentor to have another perspective.

**Review by JuanequeX — APPROVED** [review]
LGTM

**Pull request merged**